packages/primitives/src/Radio/Radio.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react'
+import { Box, StyledSystemProps } from '../Box/Box'
+import { SvgIcon } from '../Svg'
+import { Input } from '../Input'
+
+type InputProps = {
+  variant?: string
+  variantKey?: string
+  sx?: object
```

Comment:
Does this sx work as a `SxStyleProp`?

Reply:
Yep, good catch.

Reply:
Hmm actually that causes this error to pop up:

```
Exported variable 'Radio' has or is using name 'CSSSelectorObject' from external module "/Users/wescole/Jilt/kodiak-ui/node_modules/@types/styled-system__css/index" but cannot be named.
```